connect/file/src/main/java/org/apache/kafka/connect/file/FileStreamSinkConnector.java

```diff
@@ -66,4 +70,15 @@ public void start(Map<String, String> props) {
     public void stop() {
         // Nothing to do since FileStreamSinkConnector has no background monitoring.
     }
+
+    @Override
+    public ConfigDef config() {
+        if (this.configDef != null) {
```

Comment:
Referring to above comment -- this should be removed and we can replace with something like

```
private static final ConfigDef CONFIG = new ConfigDef().define(...);

@Override
public ConfigDef config() {
    return CONFIG;
}
```